src/sdk/model/advanced-backups/backup-task/backup-restore-task.ts

```diff
@@ -0,0 +1,126 @@
+import { RestoreTaskStatus } from './__JSON__/restore-task-status-enum';
+import { RestoreTaskType } from './__JSON__/restore-task-type';
+import { BackupRestoreTaskJson } from './__JSON__/backup-restore-task-json';
+import { RestoreTaskObject } from './restore-task-object';
+import { RestoreTaskObjectState } from './restore-task-object-state';
+
+/**
+ * Backup restore task.
+ */
+export class BackupRestoreTask {
```

Comment:
cory merged [this PR](http://git.iland.test/projects/IPS/repos/svc-cohesity-iaas-backups/pull-requests/38/diff#lib/recovery/recovery-api-base-impl/src/main/java/com/iland/cohesity/iaas/backups/recovery/baseimpl/model/CohesityRestoreTask.java) earlier today. would it be quick and simple enough to add the new `taskName` field to this model here? or would we prefer to wait on this big guy to be merged as is and then separately PR anything else going forward to ab-beta? only asking cuz it's tiny 😅

Reply:
Added. But there currently seems to be api side issues when running vdc.listBackupRestoreTasks() before and after model update.